scripts/fires/mapping.py

```diff
@@ -0,0 +1,224 @@
+cause_dcid_map = {
```

Comment:
please add copyright header and a top-level module comment. since this is a const - the var name should be all caps and should include a short comment

Reply:
Done